nextflow_schema.json

```
@@ -210,5 +210,11 @@
         {
             "$ref": "#/$defs/generic_options"
         }
-    ]
+    ],
+    "properties": {
+        "queries_dir": {
+            "type": "string",
```

```suggestion
            "type": "string",
            "format": "directory-path",
            "exists": true,
```
This will add some extra validation to ensure that the value is an existing directory